reference/replica-operations.md

```diff
@@ -32,45 +29,42 @@ N.B.: By application of the 2nd rule, replicas of the same volume cannot exist w
 
 A cluster has two Mayastor Nodes \(MSN\) deployed, "Node-1" and "Node-2".  Each MSN hosts two MayastorPools \(MSP\) and currently, no MayastorVolumes \(MSV\) have been defined.  MSN Node-1 hosts MSPs "Pool-1-A" and "Pool-1-B", whilst MSN Node-2 hosts "Pool-2-A and "Pool-2-B".  When a user creates a PVC from a StorageClass which defines a replica count of 2, the Mayastor control plane will seek to place one replica on each MSN (it 'follows' Rule 2).  Since in this example it can find a suitable candidate pool with sufficient free capacity on each node, the MSV is provisioned and becomes "healthy" (Rule 1).  Pool-1-A is selected on Node-1, and Pool-2-A selected on Node-2 (all pools being of equal capacity and replica count, in this initial 'clean' state).
 
-Sometime later, the physical disk of Pool-2-A encounters a hardware failure and goes offline.  The MSV is in use at the time, so its nexus \(NVMe controller\) starts to receive I/O errors for the replica hosted in that Pool.  The MSV state becomes `degraded` and the replica associated with Pool-2-A enters the `CHILD_FAULTED` state (as seen in the MSV custom resouce).
+Sometime later, the physical disk of Pool-2-A encounters a hardware failure and goes offline.  The MSV is in use at the time, so its nexus \(NVMe controller\) starts to receive I/O errors for the replica hosted in that Pool.  The associated replica's child from Pool-2-A enters the `Faulted` state and the MSV state becomes `Degraded` (as seen through the kubectl-mayastor plugin).
+
 
-Expected Behaviour:  The volume will maintain read/write access for the application via the remaining healthy replica.  The volume's status will remain `degraded` as it is not currently possible for the control plane to create a new replica - Rule 2 applies; the faulted replica is situated on Node-2 and is still a member of the volume's nexus (Rule 3), so a new replica cannot be placed here.  Even though that node has a second, unused pool, the requirement is that ALL replicas of a volume must be placed on different nodes.  There are no other MSNs in the cluster, so the selection of a new replica location fails.
+Expected Behaviour: The volume will maintain read/write access for the application via the remaining healthy replica. The faulty replica from Pool-2-A will be removed from the Nexus thus changing the nexus state to `Online` as the remaining is healthy. A new replica is created on either Pool-2-A or Pool-2-B and added to the nexus.  The new replica child is rebuilt and eventually the state of the MSV returns to `Online`.
```

Comment:
Do we go "Degraded" again at this point, until the rebuild is complete?

Reply:
Ah yes indeed, I'll highlight that

Reply:
Oh it seems to be already, just above:  MSV state becomes `Degraded` (as seen through the kubectl-mayastor plugin).